src/quo2/components/navigation/bottom_nav_tab/view.cljs

```diff
@@ -26,9 +27,12 @@
     :on-press               bottom-tab on-press function
     :pass-through?          true/false
     :icon-color-anim        reanimated shared value
+    :customization-color    Customization color for the notification mark
   "
   [{:keys [icon new-notifications? notification-indicator counter-label
-           on-press pass-through? icon-color-anim accessibility-label test-ID]}]
+           on-press pass-through? icon-color-anim accessibility-label test-ID
+           customization-color]
+    :or   {customization-color :blue}}]
```

Comment:
hmm, I would prefer no default in the component, we should add the default in colors/custom-color function instead? There should also be a defined constant with this value. I think @Parveshdhull added it recently.

Ideally we avoid adding this default value multiple times all over the codebase.

Reply:
I tried to use that constant but I get the following error from the linter:

```
WARNING: re-frame, status-im are not allowed in quo2 components

./src/quo2/components/navigation/bottom_nav_tab/view.cljs:            [status-im2.constants :as constants]
```

> we should add the default in colors/custom-color function instead? 

Yes, probably as an infrastructure change, we could do it, but out of scope for this PR IMO. :+1:

Reply:
Sure, we can make a large issue for cleaning up some of the colors pieces 👍